debugger/src/main/scala/org.apache.daffodil.debugger.dap/Compiler.scala

```diff
@@ -22,7 +22,7 @@ import cats.syntax.all._
 import java.nio.file.Path
 import org.apache.daffodil.sapi._
 
-trait Compiler {
+trait DAPCompiler {
```

Comment:
With Scala 2.13 using the name `Compiler` cashed a crash with `fs2.Compiler` which is imported in the `Parse.scala` file using `fs2._` so it needed renamed. If there is a different name we would rather use I can change it

Reply:
I think DAPCompiler is fine.